src/js/services/conversation.js

```diff
@@ -91,12 +91,13 @@ const addMessage = (text, extra = {}) => {
             _clientId: Math.random(),
             _clientSent: Date.now() / 1000,
             deviceId: getDeviceId(),
-            sendStatus: SEND_STATUS.SENDING,
-            ...extra
+            sendStatus: SEND_STATUS.SENDING
         };
 
-        if (text) {
-            message.text = text;
+        if (typeof props === 'string') {
+            message.text = props;
+        } else {
+            Object.assign(message, ...props);
```

Comment:
I guess it spreads the object as multiple objects with only one key and one value and `Object.assign` accepts multiple objects in a row